src/index.js

```diff
@@ -58,6 +58,23 @@ const handleToggles = (targets, toggleClass) => {
 
 };
 
+/**
+ * Toggles the target you have clicked, and hides all other elements in the group
+ *
+ * @param {string} target
+ * @param {string} groupTarget
+ * @param {string} toggleClass
+ */
+const handleHideAll = (target, groupTarget, toggleClass) => {
+
+    $(`[data-toggle-group=${groupTarget}]`)
+        .forEach(element => {
+            const type = element.getAttribute('data-toggle-name') === target ? 'toggle' : 'hide';
```

Comment:
Wouldn't this hide the `target` if it was already visible?

Reply:
Yes, that's expected functionality. Clicking a button will toggle (either show/hide) the target element, but close all other elements. The reason I've done this as opposed to using the existing approach of (`"data-toggle-target="ayden hide:one hide:two: hide: three"`), is due to the fact I'm dealing with dynamic card sections.

Reply:
Sorry, I didn't explain myself very well; if you clicked the only visible item in the accordion, wouldn't it hide that item as well as all of the others?

Reply:
Yep, that's correct. This is as per current functionality on the live payments page.

Reply:
OK, might be worth writing a test for that (multiple clicks hides the element again) if it's expected functionality?